app/views/layouts/application.html.slim

```diff
@@ -2,7 +2,7 @@ doctype html
 html.govuk-template lang="en"
   head
     meta content=("text/html; charset=UTF-8") http-equiv="Content-Type" /
-    title Child development training
+    title=html_title(@model&.heading, training_module_title(@training_module), module_item_title(@module_item))
```

Comment:
I'd say just accept the `module_item` because from that we can obtain both the `parent` training module and the `model`.

Reply:
Yes, I noticed that yesterday and tried to get it working, but ran into multiple issues.  Unless there is a pressing need for this to go in, I'm going to take a break from this ticket and work on some other things and come back to it fresh.